crates/ruff_python_formatter/src/expression/string/docstring.rs

```diff
@@ -609,13 +661,18 @@ impl<'src> CodeExample<'src> {
     fn add_start(
         &mut self,
         original: InputDocstringLine<'src>,
-    ) -> Option<InputDocstringLine<'src>> {
+        queue: &mut VecDeque<CodeExampleAddAction<'src>>,
+    ) {
```

Comment:
The documentation needs updating, considering that the function no longer returns the unchanged line.